action.yaml

```diff
@@ -53,4 +53,4 @@ branding:
 
 runs:
   using: "docker"
-  image: "docker://gcr.io/openssf/scorecard-action:v2.3.1"
+  image: "docker://gcr.io/openssf/scorecard-action:v2.3.2"
```

Comment:
putting a temp hold, as we may want to include the fix for https://github.com/ossf/scorecard/issues/4059